pkgs/applications/graphics/farge/default.nix

```diff
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, stdenv, fetchFromGitHub, waylandSupport ? true
```

Comment:
You don't need config and pkgs as function arguments here. config isn't used at all, and instead of pkgs, you can directly have bash, imagemagick, slurp, grim, wl-clipboard and feh as arguments.